shared/chat/conversation/list.desktop.js

```diff
@@ -169,36 +169,67 @@ class ConversationList extends Component<void, Props, State> {
     this._onScrollSettled()
   }, 100)
 
-  _showPopup (message: Message, event: any) {
-    if (message.type !== 'Text') return
-    this.setState({
-      selectedMessageID: message.messageID,
-    })
+  _renderPopup (message: Message, style: Object): ?React$Element<any> {
+    switch (message.type) {
+      case 'Text':
+        return (
+          <TextPopupMenu
+            message={message}
+            onEditMessage={this.props.onEditMessage}
+            onDeleteMessage={this.props.onDeleteMessage}
+            onLoadAttachment={this.props.onLoadAttachment}
+            onOpenInFileUI={this.props.onOpenInFileUI}
+            onHidden={() => {
+              ReactDOM.unmountComponentAtNode(document.getElementById('popupContainer'))
+              this.setState({
+                selectedMessageID: undefined,
+              })
+            }}
+            style={style}
+          />
+        )
+      case 'Attachment':
+        const {downloadedPath, filename, messageID} = message
+        return (
+          <AttachmentPopupMenu
+            message={message}
+            onDeleteMessage={() => this.props.onDeleteMessage(message)}
```

Comment:
:sheep: Can this just be `this.props.onDeleteMessage` (compared to the 'Text' branch)?